source/strings.tex

```diff
@@ -908,6 +910,8 @@
     constexpr basic_string& append(initializer_list<charT>);
 
     constexpr void push_back(charT c);
+    template<@\exposconcept{container-compatible-range}@<charT> R>
+      constexpr basic_string& append_range(R&& rg);
```

Comment:
This should probably be put up, between `append(InputIterator, InputIterator)` and `append(initializer_list<charT>)`. Both fits naturally (with iterator-pair ctor) and consistent with specification order.

Reply:
fixed